localstack/utils/kinesis/kinesis_connector.py

```diff
@@ -245,6 +245,7 @@ def _start_kcl_client_process(
     env_vars = {
         "AWS_CBOR_DISABLE": "true",
         "AWS_ACCESS_KEY_ID": account_id,
+        "AWS_SECRET_ACCESS_KEY": account_id,
```

Comment:
I think it is fine to set this here, but for this value it should be possible to set anything. Any particular reason to set it to the account id?

Reply:
For `AWS_SECRET_ACCESS_KEY`, we can set `internal secret key (__internal__)` or `account_id` here. @viren-nadkarni suggested to use account id here.